### ISSUE TITLE
Database model creation

### ISSUE THREAD

**Issue by HarshDobariya79** [HarshDobariya79/LJ-Connect-backend#7]
**Tasks:**

- Models related to university requirements to be created.
- `related_name` and `verbose_name` should be used wherever possible for backward accessibility and better admin site readability respectively.
